batch_command.go

```diff
@@ -23,25 +23,42 @@ type BatchCommand struct {
 	commands          []ICommandData
 	attachmentStreams []io.Reader
 	options           *BatchOptions
+	Result            *JSONArrayResult
 
-	Result *JSONArrayResult
+	transactionMode             int
+	disableAtomicDocumentWrites *bool
+	raftUniqueRequestId         string
```

Comment:
In go `BatchCommand` and `ClusterWideCommand` are implemented via `BatchCommand` and if requires we build request based on this properties.